app/controllers/episodes_controller.rb

```diff
@@ -10,6 +10,12 @@ def show; end
 
   def edit; end
 
+  def download
+    @episode.backup_video.attach(io: File.open(DownloadService.new(@episode).download),
```

Comment:
is it work correct after second downloading attempt?

Reply:
Yes, test it now